src/main/resources/assets/computercraft/lua/rom/programs/uptime.lua

```diff
@@ -0,0 +1,8 @@
+local function formatUptime( nTime )
+    local nHour = math.floor( nTime/60/60 )
+    local nMinute = math.floor( (nTime/60)  ) - nHour*60
+    local nSecond = math.floor( nTime ) - nMinute*60 - nHour*60*60
+    return nHour..":"..nMinute..":"..nSecond
```

Comment:
This formats time like `0:1:35`, when intuitively, it should be like `00:01:35`.